project-base/storefront/components/Blocks/OpeningHours/OpeningHours.tsx

```diff
@@ -32,39 +36,72 @@ export const OpeningHours: FC<{ openingHours: TypeOpeningHours }> = ({ openingHo
         }
     };
 
+    if (openingHours.openingHoursOfDays.length === 0) {
+        return null;
+    }
+
     return (
-        <div className={twMergeCustom('flex w-full flex-col text-sm', className)}>
-            {openingHours.openingHoursOfDays.map(({ date, dayOfWeek, openingHoursRanges }) => {
-                const isToday = openingHours.dayOfWeek === dayOfWeek;
-                const isClosedWholeDay = openingHoursRanges.length === 0;
+        <>
+            <div className="my-1 text-textDisabled">{t('Open') + ': '}</div>
+            {'exceptionDays' in openingHours && openingHours.exceptionDays?.length && (
+                <div className={twMergeCustom('flex w-full flex-col text-sm text-textError', className)}>
+                    {openingHours.exceptionDays.map((exceptionDay) => {
+                        let exceptionDayText = formatDate(exceptionDay.from, 'D.M.');
+
+                        if (exceptionDay.to) {
+                            exceptionDayText += ` - ${formatDate(exceptionDay.to, 'D.M.')}`;
+                        }
+
+                        if (exceptionDay.times.length) {
+                            for (let index = 0; index < exceptionDay.times.length; index++) {
+                                if (index === 0) {
+                                    exceptionDayText += ` ${exceptionDay.times[index].open} - ${exceptionDay.times[index].close}`;
+                                } else {
+                                    exceptionDayText += `, ${exceptionDay.times[index].open} - ${exceptionDay.times[index].close}`;
+                                }
+                            }
+                        } else {
+                            exceptionDayText += ` ${t('Closed')}`;
```

Comment:
Missing space after "Closed"
![Screenshot 2024-08-13 at 10 00 44 AM](https://github.com/user-attachments/assets/12e57528-9b01-4696-bf91-644e3c1e238c)

Reply:
Nice catch! They are actually supposed to be multiple divs (multiple lines). I fixed it.